### PR TITLE
update: added single ques/ans and results page UI

### PR DESCRIPTION
**Pull request by simran61**
Made STATIC UI pages for 
- Single page Ques 
- Single Page Ans
- Results page
Screenshot (2366)
Screenshot (2365)

results

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import SideNavBar from "./components/SideNavBar/SideNavBar";
 import Home from "./Pages/Home/Home";
 import Rules from "./Pages/Rules/Rules";
 import SingleQuestion from "./Pages/SingleQuestion/SingleQuestion";
+import SingleAnswer from "./Pages/SingleAnswer/SingleAnswer";
 import Results from "./Pages/Results/Results";
 import ProfilePage from "./Pages/ProfilePage/ProfilePage";
```
Comment (ragini07) [inline src/App.js:12]: try using barrel export

**Review by ragini07 — COMMENTED** [review]
rest of the code looks good
UI is awesome

**Review by ragini07 — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -26,6 +27,7 @@ const App = () => {
             <Route path="/" element={<Home />} />
             <Route path="/rules" element={<Rules />} />
             <Route path="/singlequestion" element={<SingleQuestion />} />
```
Comment (shraddha-1402) [inline src/App.js:29]: might use camel case when combining 2 or more words, like, singleQuestion, or might separate it with a hyphen

File: src/Pages/SingleAnswer/SingleAnswer.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
+.right {
```
Comment (shraddha-1402) [inline src/Pages/SingleAnswer/SingleAnswer.css:1]: class name could be more descriptive

**Review by shraddha-1402 — APPROVED** [review]
UI looks really good!
Code is also clean

**Pull request merged**